kratos/processes/calculate_discontinuous_distance_to_skin_process.h

```diff
@@ -204,6 +205,9 @@ class KRATOS_API(KRATOS_CORE) CalculateDiscontinuousDistanceToSkinProcess : publ
 
     Flags mOptions;
 
+    static const std::size_t mNumNodes = TDim + 1;
+    static const std::size_t mNumEdges = (TDim == 2) ? 3 : 6;
```

Comment:
```suggestion
    constexpr std::size_t NumNodes = TDim + 1;
    constexpr std::size_t NumEdges = (TDim == 2) ? 3 : 6;
```

Reply:
`constexpr` does not work as a member variable.. I don't really know why, do you know what the difference is? I only know that `constexpr` is inline implicitly and I tried `inline static const`, which works.

Reply:
OK. I've asked @RiccardoRossi and he thinks that this requires c++17. Lets live with the `static const`.